packages/cli-kit/package.json

```diff
@@ -32,7 +36,7 @@
   },
   "scripts": {
     "clean": "shx rm -rf dist",
-    "build": "rimraf dist/ && rollup -c",
+    "build": "tsc -b",
```

Comment:
this is the big change :)

Reply:
Either that or configure `tsconfig.json` to ouptut `tsconfig.tsbuildinfo` into `dist`. Like [here](https://github.com/redwoodjs/redwood/blob/main/packages/web/tsconfig.json#L6).